### ISSUE TITLE
Missing id for aria-controls attribute of the collapse

### ISSUE THREAD

**Issue by guisaas** [ng-bootstrap/ng-bootstrap#4148]
### Bug description:
In the collapse example https://ng-bootstrap.github.io/#/components/collapse/examples, aria-controls references a "collapseExample" id that does not exist in this example: 

```html
<p>
  <button type="button" class="btn btn-outline-primary" (click)="collapse.toggle()" [attr.aria-expanded]="!isCollapsed"
    aria-controls="collapseExample">
    Toggle
  </button>
</p>
<div #collapse="ngbCollapse" [(ngbCollapse)]="isCollapsed">
  <div class="card">
    <div class="card-body">
      You can collapse this card by clicking Toggle
    </div>
  </div>
</div>
```
should become:
```html
<p>
  <button type="button" class="btn btn-outline-primary" (click)="collapse.toggle()" [attr.aria-expanded]="!isCollapsed"
    aria-controls="collapseExample">
    Toggle
  </button>
</p>
<div #collapse="ngbCollapse" [(ngbCollapse)]="isCollapsed" id="collapseExample">
  <div class="card">
    <div class="card-body">
      You can collapse this card by clicking Toggle
    </div>
  </div>
</div>
```

This issue has been found using Axe: Invalid ARIA attribute value: aria-controls="collapseExample"

### Versions of Angular, ng-bootstrap and Bootstrap:

Angular: 12

ng-bootstrap: 10

Bootstrap: 4.5.3
